words/tails.txt

```diff
@@ -317,3 +317,19 @@ soay
 xalda
 ewe
 lamb
+great
+emperor
+king
+adelie
+chinstrap
+gentoo
```

Comment:
Somehow I've made it this many years before discovering that "gentoo" isn't just a Linux distro but also a kind of penguin 🤦‍♂️

Reply:
![EverythingIsConnectedNeilDegrasseTysonGIF](https://user-images.githubusercontent.com/79330/202872234-03dbdf93-a503-4f8d-83fd-d2378146a4b7.gif)